backend/test/services/release.spec.ts

```diff
@@ -342,18 +344,100 @@ describe('services > release', () => {
   })
 
   test('getModelReleases > good', async () => {
-    await getModelReleases({} as any, 'modelId')
-
-    vi.mocked(releaseModelMocks.lookup).mockImplementation(() => ({
-      ...releaseModelMocks.lookup,
+    releaseModelMocks.aggregate.mockImplementation(() => ({
+      match: vi.fn().mockImplementation(() => ({
+        sort: vi.fn().mockImplementation(() => ({
+          lookup: vi.fn().mockImplementation(() => ({
+            append: vi.fn().mockImplementation(() => ({
+              lookup: vi.fn().mockImplementation(() => [{ _id: 'release', modelId: 'test', semver: '1.0.0' }]),
+            })),
+          })),
+        })),
+      })),
     }))
+    await getModelReleases({ dn: 'user' } as UserInterface, 'modelId')
 
     expect(releaseModelMocks.match.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.sort.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.lookup.mock.calls.at(0)).toMatchSnapshot()
     expect(releaseModelMocks.append.mock.calls.at(0)).toMatchSnapshot()
   })
 
+  //test good - give good semver range with lower and upper bounds, returns valid query
+  //test fail - invalid range
+  test('getQuerySyntax > good'),
```

Comment:
I think `getQuerySyntax` shouldn't be exported so here you would need to test calling `getModelReleases with a semverQuery` and verify that the match is what you expect.

Reply:
Done